app/controllers/api/health.php

```diff
@@ -164,7 +164,7 @@
     ->action(function ($response) {
         /** @var Appwrite\Utopia\Response $response */
 
-        $response->json(['size' => Resque::size('v1-usage')]);
+        $response->json(['size' => Resque::size(Event::USAGE_QUEUE_NAME)]);
```

Comment:
Your'e missing a namespace declaration in this file, which cause the tests to fail.

Please add constants for all the other `Resque::size` calls in this controller.

Reply:
Done 👍